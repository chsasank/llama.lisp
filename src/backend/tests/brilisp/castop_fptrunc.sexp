;test for fptrunc double 1.0E+300 to half not done
(brilisp
    (define ((fprint float) (n float)))

    (define ((main void))
        (set (a double) (const 16777217.0))
        (set (f_n float) (fptrunc a float))
        (set (tmp float) (call fprint f_n))
        (ret)))
