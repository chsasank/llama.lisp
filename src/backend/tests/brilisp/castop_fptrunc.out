16777216.000000
